### DISCUSSION TITLE
Is not working for me

### DISCUSSION THREAD

**Discussion by Narii2623** [LTmadness/ValheimBetterServerConfig#39] [General]
Hello, I'm hosting a dedicated server on my own machine, and when I installed this it wasn't allowing me to connect to the server.

**Not sure if it is a conflict with another plugin, this is what I have installed** (I've removed BetterServerConfig for now so I could launch the server)

ASharpPen-Custom_Raids
DiscordConnector
LetMeSleep
LiteDB
Newtonsoft.Json
Valheim.DisplayBepinExInfo

**This is my config file (with the password censored)**

## Plugin GUID: org.ltmadness.valheim.betterserverconfig

[Advanced settings]

## Number of players allowed in the server, minimum 1, only works on steam browser so far
# Setting type: Int32
# Default value: 10
Server size = 10

## You can choose your server name color, doesn't work in steam server browser eg.: red
# Setting type: String
# Default value: 
Server name color = blue

## Should your server name be writen in italics, doesn't work on steam server browser
# Setting type: Boolean
# Default value: false
Server name italic = false

## Should your server name be writen in bold, doesn't work on steam server browser
# Setting type: Boolean
# Default value: false
Server name bold = true

## If empty world name will be used
# Setting type: String
# Default value: 
Steam Map name = 

## Number of backups you wanna keep
# Setting type: Int32
# Default value: 5
Number of backups = 5

## Used as sender name when using say/yell commands in server console
# Setting type: String
# Default value: Server
Server username = Server

## Should server announce in game chat when the world save is happening
# Setting type: Boolean
# Default value: false
Announce saves = false

## Text that will be shown as Game in Steam Browser, changing this does not affect under which game server will apear
# Setting type: String
# Default value: Valheim
Game Description = Valheim

[Console settings]

## Show what eveyone shouts (/s) in console
# Setting type: Boolean
# Default value: true
Show shout chat = true

## Show all chat in console, overwrites show chat shout option
# Setting type: Boolean
# Default value: true
Show chat = true

## List of commands allowed to use by mods
# Setting type: String
# Default value: kick,say,save,sleep
Commands Allowed for Mods = kick,say,save,sleep

## The size opf page when using help
# Setting type: Int32
# Default value: 10
Help Page Size = 10

[General settings]

##  it doesn't work in steam server browser, also you can find more color names here: https://www.w3schools.com/colors/colors_names.asp
# Setting type: String
# Default value: My Server Name
Server Name = <color=BLUE>Aesir</color>

# Setting type: Int32
# Default value: 2456
Server Port = 2456

# Setting type: String
# Default value: ServerWorld
World Name = DedicatedServer

## server password, please change
# Setting type: String
# Default value: changeMe
Server password = CENSORED

## Server data save location
# Setting type: String
# Default value: 
Save location = 

## Will server be publicly visable in server list
# Setting type: Boolean
# Default value: true
Publicly visable: = true

[Logger settings]

## Lowest level of logs to be recored to file (Info, Debug, Error)
# Setting type: LoggerLevel
# Default value: Info
# Acceptable values: Info, Debug, Error
Logging level = Info

**Side note that might inform the fix in some way: the default valheim dedicated server console window loaded in, one that I could not type anything into and could only use ctrl+shift+c to stop the server. My understanding is that it should've been a bash I could type directly into.**

Thank you in advance :)

**Comment by Narii2623** [comment]
Not sure what happened with the formatting there, sorry for the eye sore :P

**Comment by LTmadness** [comment]
it seems that since I worked on this last there have been changes on how server initialisation code work in Valheim, having few other thinks I need to focus on rn but will try to write an update as soon as possible( hopefully by the end of this month)

**Reply by Narii2623** [reply]
Ok thank you! :)

**Comment by Narii2623** [comment]
Hello! Do you think you'll be updating this? All good if not I understand, I really liked using it in the past. Or if you're aware of another one that does something similar I'd love to know it's name. Thanks for your work!

**Reply by LTmadness** [reply]
I will be releasing small update soon,  but it might not fix your problem, as most likely your problem is caused by 2 patches that are left in your bepinex patches folder form old version of bepinex

**Reply by Narii2623** [reply]
Hi, thanks for that. If I understand correctly, does that mean I could try deleting a couple files to fix it?

**Reply by LTmadness** [reply]
could you drop me a message on discord, I can send you my test version of latest changes as I found few things that also could be a cause of your problem

**Reply by Narii2623** [reply]
hi sorry just saw the notification now. where can i find your discord info?

**Comment by LTmadness** [comment]
hey, sorry was busy with moving, will try to look into it this weekend

**Comment by LTmadness** [comment]
I'm currently unavailable, for few more days, but is just my username

**Reply by Narii2623** [reply]
ok thanks. sent you a friend request. dm me whenever you're free :)
